Missionframework/stringtable.xml

```diff
@@ -7316,6 +7358,35 @@
             <German>KP Spielermenü v1.0.2\nvon Wyqer</German>
             <Spanish>Menú de Jugador KP v1.0.2\npor Wyqer</Spanish>
             <Czech>KP Player Menu v1.0.2\nby Wyqer</Czech>
+        </Key>
+		
+		<Key ID="STR_SECONDARY_MISSION3">
+            <Original>Humanitarian Aid</Original>
+            <French>Recherche et Sauvetage</French>
+            <German>Helikopterbergung</German>
+            <Spanish>Búsqueda y rescate</Spanish>
+            <Russian>Найти и Спасти</Russian>
+            <Italian>Cerca e Soccorri</Italian>
+            <Chinesesimp>搜索与营救</Chinesesimp>
+            <Chinese>搜索與營救</Chinese>
+            <Turkish>Ara ve kurtar</Turkish>
+            <Portuguese>Busca e Resgate</Portuguese>
+            <Korean>수색 및 구출</Korean>
+            <Czech>Záchranná mise</Czech>
+        </Key>
+        <Key ID="STR_SECONDARY_BRIEFING3">
+            <Original>&lt;t size='1.3' color='#ffa000'&gt;HUMANITARIAN AID&lt;/t&gt;&lt;br/&gt;&lt;br/&gt;&lt;t size='1'&gt;A nearby settlement has requested supplies, head out and deliver them to the highlighted settlement.&lt;/t&gt;&lt;br/&gt;&lt;br/&gt;&lt;t size='1'&gt;This will help win hearts and minds in the local region and ultimately help limit any locals engaging us in open and guerilla combat.&lt;/t&gt;&lt;br/&gt;&lt;br/&gt;&lt;t size='1.15' color='#00a0ff'&gt;Cost: 5&lt;img image='\A3\Ui_f\data\GUI\Cfg\Ranks\general_gs.paa'/&gt;&lt;/t&gt;&lt;br/&gt;&lt;t size='1.15' color='#ffa000'&gt;Reward: 5% increase in Civil Reputation.&lt;/t&gt;&lt;br/&gt;&lt;br/&gt;&lt;t size='10'&gt;&lt;img image='res\secondary\fob_obj.jpg/&gt;&lt;/t&gt;</Original>
```

Comment:
There's an apostrophe missing at the very end for the English localisation.

It should read
`image='res\secondary\fob_obj.jpg'/&gt;&lt;/t&gt;`

Currently it is
`image='res\secondary\fob_obj.jpg/&gt;&lt;/t&gt;`